webpack.config.production.js

```diff
@@ -1,46 +1,55 @@
+const path = require("path");
 const webpack = require("webpack");
 const { CleanWebpackPlugin } = require("clean-webpack-plugin");
-const HtmlWebpackPlugin = require("html-webpack-plugin");
 const { WebpackManifestPlugin } = require("webpack-manifest-plugin");
 const ZipPlugin = require("zip-webpack-plugin");
 
 const devWebpackConfig = require("./webpack.config");
 
-const isPrerendering = process.env.SCRIVITO_PRERENDER;
-
 function webpackConfig(env = {}) {
+  const { isPrerendering } = env;
+
   const {
     mode: _devMode,
     target: _devTarget,
     devServer: _devServer,
     entry: devEntry,
+    output: devOutput,
     plugins: devPlugins,
     ...sharedConfig
   } = devWebpackConfig({ ...env, production: true });
 
   const plugins = [
     new CleanWebpackPlugin(),
     ...filterDevPlugins(devPlugins),
-    new ZipPlugin({ filename: "build.zip", path: "../", pathPrefix: "build/" }),
+    new ZipPlugin({
+      filename: "build.zip",
+      path: "../",
+      pathPrefix: "build/",
+      exclude: "asset-manifest.json",
+    }),
   ];
 
-  if (isPrerendering) {
+  if (process.env.SCRIVITO_PRERENDER) {
     plugins.push(
-      new HtmlWebpackPlugin({
-        filename: "_prerender_content.html",
-        template: "_prerender_content.html",
-        chunks: ["prerender_content"],
-      })
+      new WebpackManifestPlugin({ fileName: "asset-manifest.json" })
     );
   }
 
   return {
     ...sharedConfig,
     mode: "production",
-    target: ["web", "es5"],
+    target: isPrerendering ? "node" : ["web", "es5"],
     entry: isPrerendering
-      ? { prerender_content: "./prerender_content.js", ...devEntry }
+      ? { prerender_content: "./prerender_content.js" }
       : devEntry,
+    output: isPrerendering
+      ? {
+          ...devOutput,
+          path: path.join(__dirname, "buildPrerendering"),
```

Comment:
I vote to pick a different name, that is more different to `buildPrerendered`. At least I accidentally stared a server in `buildPrerendering` on port 8080 to see, if my prerendering actually worked.

Some suggestions: `buildNodeProcessing` or `buildIntermediateProcessing`.

Reply:
`buildNode` maybe?